### DISCUSSION TITLE
The using process problem about ivadomed

### DISCUSSION THREAD

**Discussion by kenny30725** [ivadomed/ivadomed#1005] [Q&A]
Excuse me, I have some problem with below image
![2021-12-13 17-19-45 的螢幕擷圖](https://user-images.githubusercontent.com/89126940/145785158-05614b75-ebe4-40a5-8d17-287ee0c587c0.png)

and my dataset path is : 
![2021-12-13 17-28-52 的螢幕擷圖](https://user-images.githubusercontent.com/89126940/145786567-75f54f1d-c891-40b2-ad75-92e5a59ce322.png)

How can i solve it ?

**Comment by dyt811** [comment]
Hi, @kenny30725 could you double check to make sure that you are referring to the data path correctly? 

From the screenshot your path is somewhere `//???/ivadomed/path/to/bids/data/dataset/` is that what you have provided to the configuration file/json? 

Can you provide the full path from both 1) where the current folder is located, 2) as well as the configuration file here so we can double check?

**Reply by kenny30725** [reply]
This is my full path and which file is in the folder  (/home/cwlab913/ivadomed/path/to/bids/data/dataset)
![2021-12-14 15-12-55 的螢幕擷圖](https://user-images.githubusercontent.com/89126940/145950438-c00daf6a-a27a-4d6a-af40-9265b81eae3e.png)

and this is the config file/json from
![2021-12-14 15-20-12 的螢幕擷圖](https://user-images.githubusercontent.com/89126940/145951654-87dd989b-413a-44fc-8faf-6e1f2c647a81.png)

and when I input the command <<ivadomed --train -c path/to/config_file.json --path-data path/to/bids/data/dataset --path-output path/to/output/directory>>

the error is :
![2021-12-14 15-23-09 的螢幕擷圖](https://user-images.githubusercontent.com/89126940/145951756-73b5a91b-80f7-4299-a1c7-0d33052a36d0.png)
